modules/Hoot/ui/managePanel/TaskingManagerPanel.js

```diff
@@ -26,7 +26,8 @@ export default class TaskingManagerPanel extends Tab {
             options: [
                 { deriveType: 'Adds only', description: `Add all data from ${Hoot.config.secondaryLabel} in the specified area into ${Hoot.config.referenceLabel}` },
                 { deriveType: 'Cut & Replace', description: `Replace the ${Hoot.config.referenceLabel} data in this region with ${Hoot.config.secondaryLabel} data` },
-                { deriveType: 'Differential', description: `Add the features from ${Hoot.config.secondaryLabel} that don't exist in ${Hoot.config.referenceLabel}` }
+                { deriveType: 'Differential', description: `Add the features from ${Hoot.config.secondaryLabel} that don't exist in ${Hoot.config.referenceLabel}` },
+                { deriveType: 'Differential w/ Tags', description: `Add the features from ${Hoot.config.secondaryLabel} that don't exist in ${Hoot.config.referenceLabel} including tags` }
```

Comment:
"including tags" could probably more accurately be "but allow tag modifications of existing"